### PR TITLE
Improve audio connection stability

### PR DESCRIPTION
**Pull request by vrnagy**
This change improves the connection handling of audio and adds extra info abot the current connection status to the player.
Along with #149 our connection issues went away. Every people can hear the others and no reconnection issues after every game

- Fix WebRTC connection handling.
- Add extra info about connection state for players.
- Handle basic WebRTC errors.

### CODE REVIEW

**Comment by MattJustMatt** [comment]
Seeing a teal circle on some player (person isn't present in audioConnected array?). It's not clear to a user what this would mean IMHO outside of reading the code.

**Comment by MattJustMatt** [comment]
(and tbh even from reading the code I'm still not sure what to do to fix!)

**Comment by vrnagy** [comment]
It meens that there is a problem with the WebRTC connection between the Player and You, but You're right, we shoild give some description what it means

**Comment by Soitora** [comment]
Would this fix some users not being able to hear others? Or does this need #149 merged as well?

**Comment by tystuyfzand** [comment]
> It meens that there is a problem with the WebRTC connection between the Player and You, but You're right, we shoild give some description what it means

So, a warning? The universal status color for that is yellow, which may say "your connection isn't active" better than teal.

**Comment by vrnagy** [comment]
@Soitora it resolves the issue at the game end when everyone reconnects to each other. To fix the hearing problem it needs #149 as wel.

@tystuyfzand replaced the color to yellow

File: src/renderer/Voice.tsx [hunk 1]
```diff
@@ -469,7 +469,7 @@ const Voice: React.FC = function () {
 						return (
 							<Avatar key={player.id} player={player}
 								talking={!connected || !audio || otherTalking[player.id]}
-								borderColor={connected ? (audio ? '#2ECC71' : '#4fd9e0') : '#C0392B'}
+								borderColor={connected ? (audio ? '#2ECC71' : '#FFFF00') : '#C0392B'}
```
Comment (tystuyfzand) [inline src/renderer/Voice.tsx:472]: The double ternary here seems a bit dirty, perhaps it's time to split it into a variable/if statement above the return block?

**Pull request merged**